src/ServiceDescription/Lightspeed-Retail-2016.25.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+return [
+    'name'        => 'LightspeedRetail',
+    'description' => 'Lightspeed Retail API',
+    'baseUri'     => 'https://api.merchantos.com/API/',
+    'operations'  => [
+
+        /**
+         * --------------------------------------------------------------------------------
+         * CUSTOMER RELATED METHODS
+         *
+         * DOC: http://developers.lightspeedhq.com/retail/endpoints/Account-Customer/
+         * --------------------------------------------------------------------------------
+         */
+
+        'GetCustomers' => [
+            'httpMethod'    => 'GET',
+            'uri'           => 'Account/{accountID}/Customer.json',
+            'responseModel' => 'GenericModel',
+            'data'          => [
+                'root_key' => 'Customer',
+            ],
+            'parameters' => [
+                'limit' => [
+                    'location' => 'query',
+                    'required' => false,
+                    'type'     => 'integer',
+                ],
+                'offset' => [
+                    'location' => 'query',
+                    'required' => false,
+                    'type'     => 'integer',
+                ],
+                'archived' => [
+                    'location' => 'query',
+                    'required' => false,
+                    'type'     => 'boolean',
+                ],
+                'timeStamp' => [
+                    'location' => 'query',
+                    'required' => false,
+                    'type'     => 'string',
+                ],
+                'load_relations' => [
+                    'location' => 'query',
+                    'type'     => 'array',
+                    'required' => false,
+                    'filters'  => ['json_encode'],
+                ],
+                'orderby' => [
+                    'location' => 'query',
+                    'type'     => 'string',
+                    'required' => false,
+                ],
+                'additionalProperties' => [
```

Comment:
I think that additionalProperties come outside parameters key (otherwise there is no way for Guzzle to distinguish between this and a property that could indeed be called additionalProperties).